src/Backoffice/Task/App/Controllers/StoreTaskController.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace src\Backoffice\Task\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Routing\Controller;
+use src\Backoffice\Task\App\Requests\UpsertTaskRequest;
+use src\Backoffice\Task\Domain\Actions\StoreTaskAction;
+use src\Backoffice\Task\Domain\Models\Task;
+
+class StoreTaskController extends Controller
+{
+    public function __invoke(UpsertTaskRequest $request, StoreTaskAction $action, Task|null $task = null): JsonResponse
```

Comment:
```suggestion
    public function __invoke(UpsertTaskRequest $request, StoreTaskAction $action): JsonResponse
```
You will never receive a Task, right?